Tests/Security/Firewall/OAuthListenerTest.php

```diff
@@ -98,4 +103,10 @@ public function testHandleResponse()
         $this->assertEquals($response, $ret);
     }
 
+    public function testWrongTypeForOAuthListener()
```

Comment:
this test should be removed IMO. We don't guarantee we will throw such exception in the future, as we will switch back to a typehint when dropping support for Symfony 2.3